504.xml

```diff
@@ -0,0 +1,404 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- 
+FILE INFORMATION
+OMA Permanent Document
+   File: OMA-SUP-XML_504-V1_0-YYYY-MM-DD-A
+   Path: https://www.openmobilealliance.org/release/ObjLwM2M_Remote_SIM_Provisioning
+Public Reachable Information
+   Path: https://github.com/OpenMobileAlliance/lwm2m-registry
+   Name: 504.xml
+NORMATIVE INFORMATION
+  Send comments to https://github.com/OpenMobileAlliance/OMA_LwM2M_for_Developers/issues
+LEGAL DISCLAIMER
+Copyright 2020 Open Mobile Alliance. 
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions
+are met:
+1. Redistributions of source code must retain the above copyright
+notice, this list of conditions and the following disclaimer.
+2. Redistributions in binary form must reproduce the above copyright
+notice, this list of conditions and the following disclaimer in the
+documentation and/or other materials provided with the distribution.
+3. Neither the name of the copyright holder nor the names of its
+contributors may be used to endorse or promote products derived
+from this software without specific prior written permission.
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+The above license is used as a license under copyright only.  Please
+reference the OMA IPR Policy for patent licensing terms:
+https://www.omaspecworks.org/about/intellectual-property-rights/
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://openmobilealliance.org/tech/profiles/LWM2M-v1_1.xsd">
+<Object ObjectType="MODefinition">
+		<Name>SM-DP+ Remote SIM Provisioning</Name>
```

Comment:
The object name "SM-DP+ Remote SIM Provisioning" does not capture the generic use of this object for different Remote SIM Provisioning solutions. This object covers both GSMA eSIM where an SM-DP+ server is used as provisioning server but where a LwM2M server is used for profile management. The object also covers nuSIM where LwM2M server is used both for providing a profile package and for profile management operations. In future use of GSMA eSIM for low-power IoT the profile package may likely be downloaded from SM-DP+ via a LwM2M server to the device (eUICC).

Reply:
We are fine with a new name. This object should address the use case where the LwM2M client connects directly to the SM-DP+ server or the use case where the LwM2M server is an intermediary between the LwM2M client and the SM-DP+ server. For the use case where the LwM2M server is the repository of the SIM Update Package and is used to download that package to the LwM2M client where the update SIM process is locally executed is addressed in Object 505.

Reply:
Since this object and its resources is handling GSMA Remote SIM Provisioning use cases and behavior we suggest to call this object "GSMA Remote SIM Provisioning" with the following description:
"This is a device management object that should be used for Remote SIM Provisioning according to GSMA standards".

Reply:
Changed the name and description as suggested.